_rules/image-non-empty-accessible-name-23a2a8.md

```diff
@@ -59,6 +59,8 @@ _There are currently no assumptions._
 
 ## Background
 
+Some elements have a role of `button` and a default accessible name defined by the [HTML Accessibility API Mapping][html aam input button], for example `input` elements whose `type` [attribute value][] is either `submit` or `reset`. This rule consider that these default names can be descriptive and therefore does not fail them.
```

Comment:
I'm confused why this change is being made to `image-non-empty-accessible-name` instead of being made to `Button has non-empty accessible name`. I don't think this rule references any buttons. 

I think the statement `This rule considers that these default names can be descriptive and therefore does not fail them.` should be added as an assumption to `button has non-empty accessible name` as a counter part to the assumption in `image-button has non-empty accessible name`.